logic/sendFreezeOrder.js

```diff
@@ -168,7 +168,8 @@ SendFreezeOrder.prototype.apply = async function (trs, block, sender, cb) {
 		await utils.updateDocument({
             index: 'stake_orders',
             type: 'stake_orders',
-            body: stakeOrders.prev
+            body: stakeOrders.prev,
+			id: stakeOrders.prev.stakeId
```

Comment:
stakeOrders.prev.stakeId
```suggestion
stakeOrders.prev.id
```

Reply:
I think is better use "stakeId" because it field is table's primary key.